Source/WebDriver/socket/HTTPServerSocket.cpp

```diff
@@ -117,7 +118,7 @@ String HTTPRequestHandler::packHTTPMessage(HTTPRequestHandler::Response&& respon
     StringBuilder builder;
     const char* EOL = "\r\n";
 
-    builder.append("HTTP/1.0 ", response.statusCode, ' ', response.statusCode == 200 ? "OK" : "ERROR", EOL);
+    builder.append("HTTP/1.0 ", response.statusCode, ' ', response.statusCode == httpStatusOk ? "OK" : "ERROR", EOL);
```

Comment:
So this failed the build because this patch introduced a dependency on WebCore for platforms using the Socket based inspector, aka Windows and PlayStation. Given that its just one place that this is used I would lean towards just reverting back to the previous version instead of adding a build dependency.